packages/core/crates/core-hopr/src/lib.rs

```diff
@@ -216,6 +230,21 @@ pub fn build_components(
             )
             .map(|_| HoprLoopComponents::Swarm),
         ),
+        Box::pin(async move {
+            UniversalTimer::new(Duration::from_secs(60))
+                .timer_loop(|| async {
+                    let bloom = tbf_clone.read().await.clone(); // Clone to immediately release the lock
+                    if let Err(_) = save_tbf.call1(
+                        &wasm_bindgen::JsValue::null(),
+                        js_sys::Uint8Array::from(bloom.to_bytes().as_ref()).as_ref(),
+                    ) {
```

Comment:
For future reference, it'd be better to only have the Rust closure here... and wrap the wasm_bindgen callback into that closure and pass it to this function, so that the wasm types remain mostly in the mod wasm module.